static/app/components/profiling/flamegraphSearch.tsx

```diff
@@ -134,14 +134,25 @@ function FlamegraphSearch({
     [canvasPoolManager]
   );
 
+  React.useEffect(() => {
```

Comment:
Should we stop importing all of `React` and just import what we need?

Reply:
100%